properties.schema

```diff
@@ -137,6 +137,15 @@
                   "help": "Controls whether the assessment should be reset automatically (up to the number of available attempts) when a user revisits the page.",
                   "validators": []
                 },
+                "_allowResetIfPassed": {
+                  "type": "boolean",
+                  "required": true,
+                  "default": false,
+                  "title": "Allow retry after assessment is passed.",
```

Comment:
Please remove full stop.

@moloko would probably recommend a question mark❓

Reply:
@moloko  would you like to have ? there ??

Reply:
i think need version bump too?

Reply:
yes to the question mark, don't worry about the version bump, I find it's easiest to sort that out at the point I actually do the release

Reply:
I would actually write this as "Allow the learner to retry the assessment even if they have already passed it."